src/api-calls.js

```diff
@@ -21,4 +18,61 @@ const checkResponseStatus = (response) => {
     }
 }
 
+const cleanAllMovies = ({ movies }) => {
+    const cleanedMovies = movies.map(movie => {
+        return {
+            id: movie.id,
+            key: movie.id,
+            title: movie.title,
+            poster_path: movie.poster_path,
+            average_rating: movie.average_rating,
+        }
+    })
+
+    return cleanedMovies
+}
+
```

Comment:
Great job cleaning the data! I want to see how we can make this function more SRP without having random methods in our API Calls.